### PR TITLE
Umbraco on azure part 2

### PR DESCRIPTION
**Pull request by ghost**
blog post - scaling umbrac_- ii

### CODE REVIEW

File: _posts/2016-06-24-scaling-umbraco-on-windows-azure-II.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+name: scaling-umbraco-on-windows-azure 
+title: Scaling Umbraco On Windows Azure Part II
+date: 2016-07-03 13:02:00 +00:00
+author: Halima Koundi
+image:
+   src: /assets/img/custom/blog/2016-06-20-umbraco-on-azure/umbraco-on-azure.jpg
+tags:
```
Comment (GiuliaMantuano) [inline _posts/2016-06-24-scaling-umbraco-on-windows-azure-II.md:11]: Is it possible to add the category? I'd add "technology"

File: _posts/2016-06-24-scaling-umbraco-on-windows-azure-II.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+- cloud 
+- windows-azure 
+- cms
+- umbraco
+---
+
+In this second post of the three parts series about how to scale an Umbraco website on Windows Azure, we will install the CMS on our cloud platform.
+
+
+### A one-click process, maybe few more clicks
```
Comment (dhatanian) [inline _posts/2016-06-24-scaling-umbraco-on-windows-azure-II.md]: a few

File: _posts/2016-06-24-scaling-umbraco-on-windows-azure-II.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+Add the  App settings.
+
+{% img /assets/img/custom/blog/2016-06-20-umbraco-on-azure/08.png "" %}
+
+We're all set up. Click create, your app is being deployed.
+
+{% img /assets/img/custom/blog/2016-06-20-umbraco-on-azure/09.png "" %}
+
+After few minutes the deployment is complete.
```
Comment (dhatanian) [inline _posts/2016-06-24-scaling-umbraco-on-windows-azure-II.md:49]: a few

File: _posts/2016-06-24-scaling-umbraco-on-windows-azure-II.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+{% img /assets/img/custom/blog/2016-06-20-umbraco-on-azure/14.png "" %}
+
+### Ready to use layout template
+
+Since we installed the default template along with the Umbraco install, we have a starter kit, named Fanoe with layout template and site structure already built for us.
+
+{% img /assets/img/custom/blog/2016-06-20-umbraco-on-azure/15.png "" %}
+
+We can study, in the admin of the website accessed via the ```/umbraco path``` , the structure of our website.
+We have a Homepage that is our Higher level page, and it contains ```Learn, Explore and Extend``` as children items.
```
Comment (dhatanian) [inline _posts/2016-06-24-scaling-umbraco-on-windows-azure-II.md:78]: child items

**Pull request merged**